src/js/electron/tron/windowManager.ts

```diff
@@ -67,7 +77,10 @@ export default function windowManager() {
         this.getWindows().map((w: BrimWindow) => w.confirmClose())
       )
         .then((oks) => oks.every((ok) => ok))
-        .catch(() => false)
+        .catch((e) => {
+          log.error(e)
```

Comment:
This is where the error was silently being handled. We now log it, and say we're ok to quit. Better to allow the quit, than to hang.